### PR TITLE
Pipeline pr to homebrew

### PR DESCRIPTION
**Pull request by magodo**
Pipeline will now create a pr to homebrew to upgrade the formula for new version.

Related with #143.

### CODE REVIEW

**Comment by magodo** [comment]
@koudaiii Would you like to review this PR?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -630,3 +631,29 @@ stages:
...
+              # Install brew
+              export NONINTERACTIVE=1
+              /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+              export PATH=$PATH:/home/linuxbrew/.linuxbrew/bin
+
+              # Bump formula
+              cd "$(brew --repository homebrew/core)"
+              git config user.name "azure-terraform-bot"
+              git config user.email "azure-terraform@microsoft.com"
+              HOMEBREW_GITHUB_API_TOKEN=${PAT} brew bump-formula-pr --version=${VERSION:1} --no-browse --force aztfy
```
Comment (koudaiii) [inline azure-pipelines.yml:655]: I checked https://github.com/Homebrew/homebrew-core/pull/112713/files
And, I confirmed the manual test.

```console
$ brew bump-formula-pr --version=0.9.0 --no-browse --force aztfy --dry-run
Warning: These pull requests may be duplicates:
aztfy 0.9.0 https://github.com/Homebrew/homebrew-core/pull/112713
aztfy 0.9.0 https://github.com/Homebrew/homebrew-core/pull/112712
aztfy 0.9.0 https://github.com/Homebrew/homebrew-core/pull/112711
==> Cloning https://github.com/Azure/aztfy.git
Updating /Users/sakabekodai/Library/Caches/Homebrew/aztfy--git
From https://github.com/Azure/aztfy
 * [new tag]         v0.9.0     -> v0.9.0
==> Checking out tag v0.9.0
Previous HEAD position was cd5e46e update schema mapping (#106)
HEAD is now at 8d89e22 Pipeline PR to brew
HEAD is now at 8d89e22 Pipeline PR to brew
==> replace /tag:(\s+")v0.8.0(?=")/ with "tag:\\1v0.9.0\\2"
==> replace "a7c179f0a150fb5ad63206532ea891c1dc0c87f1" with "8d89e228b5a74838721d7f599045fa73ae712fbd"
==> brew audit --formula aztfy.rb
==> try to fork repository with GitHub API
==> git add /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/aztfy.rb
==> git checkout --no-track -b bump-aztfy-0.9.0 origin/master
==> git commit --no-edit --verbose --message='aztfy 0.9.0' -- /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/aztfy.rb
==> git push --set-upstream FORK_URL bump-aztfy-0.9.0:bump-aztfy-0.9.0
==> git checkout --quiet -
==> create pull request with GitHub API (base branch: master)
```

**Review by koudaiii — APPROVED** [review]
LGTM 😄

**Comment by magodo** [comment]
@koudaiii Thanks! The tag v0.9.0 is actually not finalized, which will be removed later. I made this temporary tag only for testing purpose. With this PR merged, you do not need to manually update the formula for each release. Really thank you for doing this in the past!

**Comment by koudaiii** [comment]
@magodo Thanks, too 😄

**Pull request merged**